client/signup/steps/import/types.ts

```diff
@@ -31,3 +31,22 @@ export type FeatureName =
 	| 'plugins';
 
 export type FeatureList = { [ key in FeatureName ]: string };
+
+// List of supported importer platforms (most important)
+export type ImporterMainPlatform =
+	| 'blogger'
+	| 'medium'
+	| 'squarespace'
+	| 'wordpress'
+	| 'wix'
+	| ImporterPlatformOther;
+// List of supported importer platforms (others)
+export type ImporterPlatformOther =
+	| 'blogroll'
+	| 'ghost'
+	| 'livejournal'
+	| 'movabletype'
+	| 'tumblr'
+	| 'xanga';
+export type ImporterPlatformExtra = 'godaddy-central';
+export type ImporterPlatform = ImporterMainPlatform | ImporterPlatformOther | ImporterPlatformExtra;
```

Comment:
Do we still need [Importer type](https://github.com/Automattic/wp-calypso/blob/a59a82602b8ff3beec6bd66a039b786124b31fae/client/signup/steps/import-from/types.ts#L1) since you added ImporterPlatform?

Reply:
Maybe we should use `type Importer` as the base type and not this new one? I think that beginning to take full advantage of the TS system is good. I tried to have a base point where we have all the supported platforms to add clarity and not miss something when a fresh new platform is added in the future.

Reply:
In some places platform names are somehow hardcoded, for example: [client/my-sites/migrate/step-source-select.jsx](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/migrate/step-source-select.jsx#L45) or [client/signup/steps/import/ready/platform-details.tsx](https://github.com/Automattic/wp-calypso/blob/trunk/client/signup/steps/import/ready/platform-details.tsx#L15).

Reply:
Agree with you, we can keep using the Importer type.
Thanks for the explanation!